src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Navigation from './components/header/NavBar';
+import Rockets from './components/rockets/Rockets';
+import Missions from './components/missions/Missions';
+import MyProfile from './components/myprofile/MyProfile';
+import './App.css';
+
+function App() {
+  return (
+    <Router>
+      <div>
+        <Navigation />
+        <div className="header-line" />
+        <Switch>
+          <Route exact path="/" component={Rockets} />
+          <Route path="/rockets" component={Rockets} />
+          <Route path="/missions" component={Missions} />
+          <Route path="/my-profile" component={MyProfile} />
+        </Switch>
```

Comment:
- [ ] Dear Team, You have done a great job fetching and rendering data from the endpoint.  The one thing you need to keep in mind is that data should only be rendered on an initial page load. once we are on the rocket page, we fetch the data for `rockets` then when only switch to the `mission` route the data should be fetch for the missions, but after this point switching between links/pages should not trigger a data fetch. In your case going back and forth between rocket, mission, and myprofile section triggers a data fetch. Please make sure data is only loaded once for both sections. 

### The current behavior  

https://github.com/DevKaranJ/space-travelers-hub/assets/69077061/f30973e6-837e-4cce-ab29-113627e4ae27

Reply:
The issue has been properly resolved
<img width="1292" alt="Screenshot 2023-10-21 at 11 07 22" src="https://github.com/DevKaranJ/space-travelers-hub/assets/58692590/bc19acfc-ecbe-4ae0-8972-2358215905a9">

Reply:
Hello @Whoistolu , 

Thank you for the feedback. However, I do not see if the PR is approved..